calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -7,7 +7,7 @@
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.25",
+      "minor_version": "v3.26",
```

Comment:
@hjiawei We don't need to update 3.18 now because part of the steps for publishing "next" version is to copy the array from the previous version as starting point. But I appreciate that you are tracking these details.

Reply:
I mean the calico-private repository [here](https://github.com/tigera/calico-private/blob/release-calient-v3.18/node/Makefile#L45). It is still saying `CALICO_VERSION=v3.24.5` for CE v3.18+ whereas docs is updating to `v3.26`. I believe the outdated `CALICO_VERSION` will be reported back to Tigera Manager. I understand docs don't have v3.18 versioned folder yet.

Reply:
I agree that above is true. I'm just saying that all 3.18 doc release versions are updated when 3.18 goes early preview/GA. Not before then. We have a playbook that we follow for a release. We do not update "next" versions when 3.17.1 is released.